<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedThumbnailFilter</class>
 <widget class="QDialog" name="AdvancedThumbnailFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>222</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>222</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced Thumbnail Filter</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>163</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apply filter?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="loadSavedFilterDropdown">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="toBeSetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>212</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Advanced filter to apply to thumbnails:</string>
   </property>
  </widget>
  <widget class="QLabel" name="loadSavedFilterLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>93</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load saved filter:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFilterButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveFilterDescription">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Filter nickname</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveFilterLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>130</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save filter for later use?</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="filterToApply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::WidgetWidth</enum>
   </property>
   <property name="placeholderText">
    <string>Example: view = '#010101' AND dcterms_created &lt; '2010'</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loadSavedFilterDropdown</tabstop>
  <tabstop>doneButton</tabstop>
  <tabstop>filterToApply</tabstop>
  <tabstop>saveFilterDescription</tabstop>
  <tabstop>saveFilterButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
